### ISSUE TITLE
IPC map improvements

### ISSUE THREAD

**Issue by zackarno** [OCHA-DAP/hdx-signals#165]
looking at the below for reference.

<img width="354" alt="image" src="https://github.com/OCHA-DAP/ds-aa-lac-dry-corridor/assets/23731112/c0a9bca5-2e4e-480a-ab46-3cef65d9e927">

a few points that I think would make map more appealing:

1.  Make phase title: "IPC phase" 
2. remove black dots from phase legend and make phase legend key rectangles have border
3. Does phase 2 color represent everything below 3? if so perhaps legend label should be <3 or ≤2 ? If phases 1 & 2 are supposed to be visualized (i'd prefer this) I'd make sure all factor legends are present on map. I think keeping all phases is better as people are used to that color ramp and we are making it clear that we only trigger on 3+
4. The phase 2 color looks basically transparent  in this example -- I wouldn't use any transparent colors?
5. I'd consider adding the IPC phase level classification definitions in the legend or at least somewhere in email (1 - acceptable, 2 - alert, 3 - critical ...)

In this particular map the lining up of the phase  and admin boundaries looks a bit odd, but not sure if thats actually an alignment issue or if the grey slivers are other LHZs/zones w/ phase <3

**Comment by caldwellst** [comment]
Just to note: we DO NOT want to show all 5 phases in the map legend, however, because the map legend does serve the purpose of showing the range of phase classifications in this analysis.

**Comment by zackarno** [comment]
noting 2 more things:

-  The settlement and phase legend keys are not aligned.
-  This IPC legend doesn't seem to be nice on landscape maps where legend is on bottom. Need to incorprate custom themeing


I would have thought you wanted it more like this (but aligned, and idk how to get the key not grey for settlement type yet). One color bar for both points and polys. Shapes would need to be separate?

![image](https://github.com/OCHA-DAP/hdx-signals/assets/23731112/449f9ef3-bdaa-458d-b4a4-47a38e8a06df)

**Comment by zackarno** [comment]
no idea how to get those legends aligned though. I didn't see any suggestions, but using additional libraries and still not straightforward

**Comment by caldwellst** [comment]
So annoying about the legends! Really weird. We don't want all levels plotted, though. We ONLY want the levels seen in the dataset. Because the legend is actually telling everyone what phase classifications are in this map. Otherwise, you have to really scan the map to understand what classifications have been made.

**Comment by zackarno** [comment]
ok let's not deal with the legend issue in the open PR #163 which was started mainly to address lac labels that way we can keep the discussions separate, merge that one quickly, and address this one once it's clear and we have a good solution.

I feel like there is something I'm not understanding about the desired output for this  map, nonetheless happy to try to construct as you like as you've given it more thought and it's been reviewed, but in IMO:

- for categorical map with only 5 categories it's better to show all categories especially in this case where the color break for signaling is so obvious and where we are not really giving much background in the email on IPC scores. I think it looks very odd to have a legend that says IPC Scores and then gives one orange square that says 3.

Also can you remind me how the settlement classifications (points/shapes) are worked into the actual alert mechanism (or just point to doc? there used to be readmes?)? If settlement classifications are not directly built in I would def consider removing there coloring as it will be confusing/misleading to alot of people without additional explanations - this would also make legend issue simpler

**Comment by caldwellst** [comment]
Sorry Zack, was so busy at the workshop and then rested my brain, so late to respond here. I definitely see your points on readability, and think we are trying to strike a balance here between explaining the IPC system and having standalone, interpretable maps for users with limited knowledge vs. streamlining the maps to assume a base level of knowledge.

I think since Signals is intended to be a product that users repeatedly use, I have been leaning towards the latter setup. In this way, I think we need to expect users to have some familiarity with the system. And with that, we can make sure information is as clear to the user as possible, such as using the legend to show what area phases are classified in the current setup.

However, what this highlights to me, maybe we want a `How to read the visuals` section in all of the dataset documentations on GitBook? See the current documentation here! https://un-ocha-centre-for-humanitarian.gitbook.io/hdx-signals

So, if you agree with that, we can have a separate issue to create these visual sections. Then, to fix this, I think what we want is to edit the shape type in the legend to be `shape = 21`, which would allow us to see the points actually have area fill? That would be simple and then allow the user to see clearly where the points have a certain class and when the polygon areas do?

**Comment by caldwellst** [comment]
Other option is to still stick with not showing points on the area phases section, and just have single color blocks for all area phases seen in the data, for both polygon and points.

**Comment by caldwellst** [comment]
Yeah, might be odd in those cases but I think should be relatively few. I checked the historical examples and can only see it happen when we have phase 0 (the rare cases where areas are given 0 population and an area phase of 0, like Abyei from the Sudan analysis ha). Otherwise, it's always a continuous drift.

And yes, agreed other than alignment then that should be good to go! I think the other change I would make is you changed the legend to say "IPC phase" and I think "Area phase" would be better since we know it's IPC/CH, and this distinguishes it as the area phase classification.
